### PR TITLE
dd: add pt_BR translation

### PR DESCRIPTION
**Pull request by Fabiokleis**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_BR/common/dd.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Cria um USB drive bootável a partir de um arquivo isohybrid (como uma archlinux-xxx.iso) e mostra o progresso:
+
+`dd if={{arquivo.iso}} of=/dev/{{usb_drive}} status=progress`
+
+- Clona um drive para outro drive com 4 MiB block, ignora erros e mostra o progresso:
+
+`dd if=/dev/{{drive_fonte}} of=/dev/{{drive_destino}} bs=4M conv=noerror status=progress`
+
+- Gera um arquivo com 100 bytes randômicos utilizando o kernel random driver:
```
Comment (navarroaxel) [inline pages.pt_BR/common/dd.md:14]: Which is most used?
```suggestion
- Gera um arquivo com 100 bytes aleatórios utilizando o kernel random driver:
```

Comment (Fabiokleis) [reply pages.pt_BR/common/dd.md:14]: both, but "randômico"  is an adaptation of the English

Comment (Fabiokleis) [reply pages.pt_BR/common/dd.md:14]: I think "aleatórios" is better

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**